sdk/spring-sdk/src/test/java/kalix/springsdk/impl/SpringSDKTestRunner.java

```diff
@@ -1,37 +0,0 @@
-/*
- * Copyright 2021 Lightbend Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package kalix.springsdk.impl;
-
-import kalix.javasdk.Kalix;
-import kalix.springsdk.action.ReflectiveActionProvider;
-import kalix.springsdk.testmodels.action.CounterSubscriber;
-import kalix.springsdk.testmodels.valueentity.Counter;
-import kalix.springsdk.valueentity.ReflectiveValueEntityProvider;
-
-import java.util.concurrent.ExecutionException;
-
-public class SpringSDKTestRunner {
```

Comment:
I remove this class that was here for local, manual tests.

For the moment, it's being replaced by  `kalix.springsdk.wriing.action.App`. The goal is to have many of those small apps and then write small tests that run then and do so assertions, see `ActionWiringTest`.